core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -1,9 +1,9 @@
 <script type="text/ng-template" id="summaryTile/exploration">
-  <md-card class="oppia-exploration-summary-tile" ng-class="{'small-width': !isWindowLarge}">
+  <md-card class="oppia-gallery-summary-tile" ng-class="{'small-width': !isWindowLarge}">
```

Comment:
Oh wait, sorry -- ignore this. Now I realize why you did it -- you need it for collections too.

Deleting this comment.